docs/source/tutorials/running_a_simulation/interactive.rst

```diff
@@ -332,6 +342,39 @@ into our disease model. We could do the following.
 This is an easy way to take an old model and toy around with new components
 to immediately see their effects.
 
+Removing components
+~~~~~~~~~~~~~~~~~~~
+
+Currently, there isn't a way to use the above approach to *remove* components from
+an existing model (without using private attributes).
+
+This is a very common use case, because often when we run an interactive simulation,
+we don't need any of the simulation's observers.
```

Comment:
I think you should remove this reference to "observers" since that is not a meaningful concept at the vivarium level. Observation exists in vivarium, but the idea of specific components for observation is only implemented in VPH.

Reply:
Ah interesting, I didn't know that. In that case, this sentence is probably a better fit for `vivarium_research` than `vivarium`.

Reply:
I just meant this sentence.